actionpack/lib/action_dispatch/http/parameters.rb

```diff
@@ -51,12 +51,12 @@ def path_parameters
       # Invoca Patch
       def strip_string_params!(value_to_strip)
         case value_to_strip
-          when String
-            value_to_strip.strip!
-          when Hash
-            value_to_strip.each{ |key, value| strip_string_params!(value) if !key.respond_to?(:to_sym) || key.to_sym.not_in?(do_not_strip_string_parameters) }
-          when Array
-            value_to_strip.each{ |value| strip_string_params!(value) }
+        when String
+          value_to_strip.strip!
+        when Hash
+          value_to_strip.each { |key, value| strip_string_params!(value) if !key.respond_to?(:to_sym) || !do_not_strip_string_parameters.include?(key.to_sym) }
```

Comment:
Ok, I went with that, split onto multiple lines since it was getting too long. And I put the defined?() check back because the warnings were getting annoying and the `!do_not_strip_string_parameters` check now makes nil/false harmless.